napari/_qt/layer_controls/qt_labels_controls.py

```diff
@@ -526,12 +526,28 @@ def _on_editable_change(self, event=None):
         event : napari.utils.event.Event, optional
             The napari event that triggered this method.
         """
+        widget_list = [
+            'pick_button',
+            'paint_button',
+            'fill_button',
+            'erase_button',
+        ]
+        widgets_to_toggle = {
+            (2, True): widget_list,
+            (2, False): widget_list,
+            (3, True): widget_list[0:1],
```

Comment:
```suggestion
            (3, True): widget_list[:1],
```
more consistent with rest of codebase